.teamcity/_self/projects/CalypsoDashboard.kt

```diff
@@ -0,0 +1,51 @@
+package _self.projects
+
+import Settings
+import _self.bashNodeScript
+
+import jetbrains.buildServer.configs.kotlin.v2019_2.Project
+import jetbrains.buildServer.configs.kotlin.v2019_2.Dependencies
+import jetbrains.buildServer.configs.kotlin.v2019_2.BuildType
+import jetbrains.buildServer.configs.kotlin.v2019_2.BuildStep
+import jetbrains.buildServer.configs.kotlin.v2019_2.BuildSteps
+import jetbrains.buildServer.configs.kotlin.v2019_2.triggers.finishBuildTrigger
+
+object CalypsoDashboard : Project({
```

Comment:
**Suggestion/Question:** Okay, kind of playing devil's advocate here... But should we move this back into the `WebApp` project for now?

I really appreciate you trying to break up the massive behemoth that the `WebApp` project has become... We could certainly use some more granular modularization.

And at the same time, this distinction feels a little weird to me. This is one of the only Calypso related things that isn't in the `WebApp` project, and it is tightly coupled to a build in the `WebApp` project.

What if we just put this in WebApp for now, and do a separate dedicated PR to try to break `WebApp` up into multiple steps?